### PR TITLE
feat: migrates getting started information

### PR DESCRIPTION
**Pull request by tdstein**

### CODE REVIEW

File: 01-getting-started/03-initializing-a-client/index.qmd [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+::: {.callout-warning}
+Keep your credentials safe. Do NOT use inline credentials if you need to share your program with others. Instead, provide your credentials via environment variables or obtain them through a secrets manager.
+:::
+
+:::{.panel-tabset group="language"}
+
+## Python
+
+```python
```
Comment (toph-allen) [inline 01-getting-started/03-initializing-a-client/index.qmd:53]:
```suggestion
```{.python}
```

etc.?

File: 01-getting-started/03-initializing-a-client/index.qmd [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+from posit import connect
+
+URL = "https://connect.example.com"
+API_KEY = "abcdefghijklmnopqrstuvwxyz123456"
+
+client = connect.Client(URL, API_KEY)
+```
+## R
+
+```r
```
Comment (toph-allen) [inline 01-getting-started/03-initializing-a-client/index.qmd:63]:
```suggestion
```{.r}
```

File: index.qmd [hunk 3]
```diff
@@ -1,98 +1,13 @@
...
-
-## R
-
-```{.r}
-library(connectapi)
-
-client <- connect()
-```
-:::
+Groups are utlized to to associate multiple users together and provide a simpler mechanism for managing content access controls.
```
Comment (toph-allen) [inline index.qmd:13]:
```suggestion
Groups consist of multiple users, and provide a streamlined way to manage content access controls.
```

Some typo fixes and light rewording

**Review by toph-allen — APPROVED** [review]
Looks good, with a few small comments for typo fixes etc.

File: 01-getting-started/02-creating-an-api-key/index.qmd [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+title: Creating an API Key
+---
+
+## Problem
+
+You need to obtain an API key to authenticate with your Connect server when using the Posit SDK.
+
+## Solution
+
+Follow the [API Keys User Guide](/user/api-keys/) to create an API key.
```
[the remark this answers is not in the document]
Comment (tdstein) [reply 01-getting-started/02-creating-an-api-key/index.qmd:11]: No, the absolute paths seem to work correctly on the staging branch in Connect.

File: 01-getting-started/03-initializing-a-client/index.qmd (hunk 1 above)
Comment (tdstein) [reply 01-getting-started/03-initializing-a-client/index.qmd:53]: It turns out that these are equivalent to each other for rendering purposes. Do you want to continue using {.python} or is just python ok?

Comment (aronatkins) [reply 01-getting-started/03-initializing-a-client/index.qmd:53]: My personal preference is without the braces; that's the style we use throughout the Connect documentation except for the rare instance where we need to provide additional attributes.

That said, this isn't a super strong position.

**Pull request merged**